todo_app/app.py

```diff
@@ -19,72 +18,50 @@ def create_app(debug=True):
-            writelog(context, doing, 'items',items)
+        for item in mongo_items:
+            items.append(Item(item['_id'], item['name'], item['idlist']))
         item_view_model = ViewModel(items)
-        #item_view_model = items
         return render_template('index.html', view_model=item_view_model)
-        # return render_template('index.html', items_list = items)
 
     @app.route('/add-item', methods = ["POST"])
     def add_todo_item():
         item_title = request.form.get('title')
-        #add_item(item_title)
-        add_trello_card(item_title)
-        context = 'app.py add-item route'
-        doing = 'add_trello_card'
-        writelog(context, doing, 'item_title',item_title)
+        add_mongo_item(item_title)
+        #context = 'app.py add-item route'
+        #doing = 'add_trello_card'
+        #writelog(context, doing, 'item_title',item_title)
         return redirect('/')
 
     @app.route('/del-item', methods = ["POST"])
     def del_todo_item():
         exis_item_title = request.form.get("selected_item")
-        #id = get_item_id_by_title(exis_item_title)
-        id = get_trello_card_id_by_name(exis_item_title)
-        #delete_item(id)
-        del_trello_card(id)
+        id = get_mongo_item_id_by_name(exis_item_title)
         context = 'app.py del-item route'
-        doing = 'del_trello_card'
+        doing = 'del_mongo_card'
+        writelog(context, doing, 'exis_item_title',exis_item_title)
+        del_mongo_item(id)
+        context = 'app.py del-item route'
+        doing = 'del_mongo_card'
         writelog(context, doing, 'id',id)
         return redirect('/')
 
     @app.route('/upd-item', methods = ["POST"])
     def upd_todo_item():
         exis_item_title = request.form.get("selected_item")
-        #exis_item = get_item_by_title(exis_item_title)
-        exis_item = get_trello_card_by_name(exis_item_title) 
-        #exis_item_status = exis_item['status']
-        exis_item_status = exis_item['idList']
+        exis_item = get_mongo_item_by_name(exis_item_title) 
+        exis_item_status = exis_item['idlist']
 
         item_status = request.form.get("status")
         item_title = request.form.get("upd_title")
-        #item_id = get_item_id_by_title(exis_item_title)
-        item_id = get_trello_card_id_by_name(exis_item_title)
+        item_id = get_mongo_item_id_by_name(exis_item_title)
         if item_title == '':
-            #item = { 'id': item_id, 'status': item_status, 'title': exis_item_title }
-            item = { 'id': item_id, 'idList': item_status, 'name': exis_item_title }
+            item = { 'id': item_id, 'idlist': item_status, 'name': exis_item_title }
         else:
-            #item = { 'id': item_id, 'status': item_status, 'title': item_title }
-            item = { 'id': item_id, 'idList': item_status, 'name': item_title }
+            item = { 'id': item_id, 'idlist': item_status, 'name': item_title }
```

Comment:
Similarly, this whole block (lines 59-62) appears to be ignored and can be deleted!